source/module_hamilt_lcao/module_gint/gint_gamma_vl.cpp

```diff
@@ -68,20 +67,20 @@ void Gint_Gamma::transfer_pvpR(hamilt::HContainer<double>* hR,const UnitCell* uc
             assert(upper_ap != nullptr);
 #endif
             double* lower_matrix = ap.get_pointer(0);
-            for(int irow=0; irow<ap.get_row_size(); ++irow)
+            for (int irow = 0; irow < ap.get_row_size(); ++irow)
             {
-                for(int icol=0; icol<ap.get_col_size(); ++icol)
+                for (int icol = 0; icol < ap.get_col_size(); ++icol)
                 {
-                    *lower_matrix++ = upper_ap->get_value(icol, irow);
+                    lower_matrix[ap.get_col_size() * irow + icol] = upper_ap->get_value(icol, irow);
```

Comment:
This position does cause performance reduction. We should use a auxiliary variable to balance performance and readability.